lib/mof/scanner.rb

```diff
@@ -144,10 +144,10 @@ def fill_queue
 	when "sint64" then @q.push [:DT_SINT64, CIM::Type.new(m)]
 	when "sint8" then @q.push [:DT_SINT8, CIM::Type.new(m)]
```

Comment:
I'd be happy to add in sint<size> and probably string and boolean.  As this is the first time we've had to do this, I didn't want to go overboard and match everything.  Right now, the only long name in use is uint32 and that's after processing 200+ modules.